src/Compilers/CSharp/Test/Emit/PDB/PDBUsingTests.cs

```diff
@@ -525,10 +525,10 @@ class A { void M() {  } }
                 options: TestOptions.DebugDll,
                 references: new[]
                 {
-                    // first aliased reference
-                    compilation1.ToMetadataReference(ImmutableArray.Create("X")),
-                    // second unaliased reference 
-                    compilation1.ToMetadataReference(),
+                // first aliased reference
```

Comment:
@basoundr here is the code which ran the formatter and generated this diff 

https://github.com/dotnet/codeformatter/blob/master/src/Microsoft.DotNet.CodeFormatting/Rules/FormatDocumentFormattingRule.cs#L40

It's pretty straight forward, just a call to Formatter.FormatAsync with no options.